src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs

```diff
@@ -99,6 +104,16 @@ public override void ExecuteCmdlet()
                 return;
             }
 
+            if (subscriptions.Count > SubscriptionLimit)
```

Comment:
@OlhaTkachenko what happens in the response if the user provides more subscriptions than 1000? Is there an exception that's thrown? I just want to make sure there isn't a way for us to call the below script in this cmdlet when the user provides some switch parameter, say `-AllSubscriptions`